### DISCUSSION TITLE
[Doc] Auto-split Hash Range：

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#19039] [Q&A]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### What issue do you find in Pulsar docs?

When a new consumer is connected, the largest region is chosen and is then split in half - the lower half will be mapped to the newly added consumer and upper half will be mapped to the consumer owning that region. Here is how it looks like from 1 to 4 consumers:
bottom：
![image](https://user-images.githubusercontent.com/49554320/209297540-6489c8d9-a8f7-4613-95a2-b4191890213e.png)

last step ：
C4 connected:
|------- C3 ------|------- C2 ------|------- C1 ------|------- C4 ------|
i think it error。

i think bottom is ok：
|------- C3 ------|------- C2 ------|------- C4 ------|------- C1 ------|

@momo-jun 

### What is your suggestion?

?

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by tisonkun** [comment]
Moved to https://github.com/apache/pulsar/discussions/categories/q-a since it's more on a question.

**Comment by momo-jun** [comment]
Hi @asafm, Happy holidays! Can you pls take a look at this question when you are back from the holidays? Thank you.

**Comment by asafm** [comment]
Thanks for spotting the error, @startjava! I have fixed it in https://github.com/apache/pulsar/pull/19063
